editor/plugins/sprite_frames_editor_plugin.cpp

```diff
@@ -154,14 +154,21 @@ void SpriteFramesEditor::_sheet_add_frames() {
 
 	int fc = frames->get_frame_count(edited_anim);
 
+	Ref<AtlasTexture> atlas_source = Object::cast_to<AtlasTexture>(*split_sheet_preview->get_texture());
+
+	Rect2 region_rect = Rect2(0, 0, 0, 0);
+
+	if (atlas_source != NULL && atlas_source->get_atlas().is_valid())
```

Comment:
We now use `nullptr` instead of `NULL` in the master branch, but for such constructor we'd typically just check `if (atlas_source && ...)`